src/api/server-routes.ts

```diff
@@ -51,19 +51,19 @@ export function buildRoutes(): express.Router {
   const router = express.Router();
 
   router.get('/limit/:limit/offset/:offset', (req: express.Request, res: express.Response) => {
```

Comment:
I don't know what you mean here, `userId` is in next line and is passed through query parameters, not directly by url...